### PR TITLE
Edited "how to train this model" and "known issues"

### PR DESCRIPTION
**Pull request by kelvin2468**
Revisions:
-Useful parameters can be found in the original repository.
Important notes:
1.	This model doesn’t provide default weight decay, user needs to add it themselves
2.	Due to huge memory use with OS=8, Xception backbone should be trained with OS=16 and only inferenced with OS=8. 
3.	User can freeze feature extractor for Xception backbone (first 356 layers) and only fine-tune decoder. Right now (March 2019), there is a problem with finetuning Keras models with BN. You can read more about it here.
Known issues
This model can be retrained check this notebook. Finetuning is tricky and difficult because of the confusion between training and trainable in Keras. See this issue for a discussion and possible alternatives.

updated #77

### CODE REVIEW

**Comment by bonlime** [comment]
@kelvin2468  
Thx for pull request!
